### PR TITLE
Add kerberos deployment

### LINKED ISSUE #140: Deploy a Kerberos domain controller in MBBox for testing
To test the MBBox with Kerberos we need to deploy the Kerberos domain controller.

## Acceptance Criteria
* Deploy Kerberos in MBBox test environment
* Kerberos is running and can be used

### PR DESCRIPTION
**Pull request by Zlopez**
This commit adds everything that is need to build the working kerberos
container and deploy it in kubernetes cluster.

Closes #140

## Proposed Changes

* Add kerberos Dockerfile
* Add kerberos configmap
* Add kerberos deployment
* Add kerberos service

## Verification Steps

* `cd components/kerberos`
* `kubectl apply -f .`
* `kubectl exec -it <kerberos pod> -- kinit` (default password is `admin`)
* No error thrown

## Additional Notes
In minikube you need to do the following to let verification steps to work:
* `minikube ssh`
* `sudo ip link set docker0 promisc on`

### CODE REVIEW

File: components/kerberos/2-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    matchLabels:
+      app: kerberos
+  template:
+    metadata:
+      labels:
+        app: kerberos
+    spec:
+      containers:
+        - name: kerberos
+          image: quay.io/zlopez/kerberos:latest
```
Comment (odra) [inline components/kerberos/2-deployment.yaml:17]: Can you change it to `quay.io/fedora/kerberos:latest`?

Comment (Zlopez) [reply components/kerberos/2-deployment.yaml:17]: I can, but I want to do it as part of test integration https://github.com/fedora-infra/mbbox/issues/147, currently there is no image in Fedora namespace

Comment (odra) [reply components/kerberos/2-deployment.yaml:17]: Ok, sounds good.

Comment (odra) [reply components/kerberos/2-deployment.yaml:17]: I've created a kerberos repository in fedora org.

**Review by odra — APPROVED** [review]

File: components/kerberos/2-deployment.yaml (hunk 1 above)
Comment (Zlopez) [reply components/kerberos/2-deployment.yaml:17]: Thanks, I will definitely use it when I will work on the #147

**Pull request merged**